iidm/iidm-scripting/src/test/groovy/com/powsybl/iidm/network/scripting/IdentifiableExtensionGroovyTest.groovy

```diff
@@ -45,18 +45,16 @@ class IdentifiableExtensionGroovyTest {
     void testStringProperty() {
         assertFalse(s.hasProperty())
         assertNull(s.greeting)
-        s.greeting = "hello"
+        s.setProperty("greeting", "hello")
         assertEquals("hello", s.getProperty("greeting"))
-        assertEquals("hello", s.greeting)
     }
 
     @Test
     void testBooleanProperty() {
         assertFalse(s.hasProperty())
         assertNull(s.ok)
-        s.ok = true
-        assertEquals("true", s.getProperty("ok"))
-        assertEquals(true, s.ok)
+        s.setBooleanProperty("ok", true)
```

Comment:
Revert changes and change IdentifiableExtension.groovy

Reply:
Missing previous line adapted : `assertEquals(true, s.getBooleanProperty("ok"))`
Keep previous line as : `assertEquals(true, s.ok)` and remove `assertTrue(s.ok)`